### PR TITLE
fix usestore update & destroy

### PR DESCRIPTION
**Pull request by seb-odoo**

### CODE REVIEW

**Comment by seb-odoo** [comment]
Ready for me, requesting review on these changes

File: src/store.ts [hunk 1]
```diff
@@ -95,37 +95,42 @@ export function useStore(selector, options: SelectorOptions = {}): any {
   if (!store.updateFunctions[component.__owl__.id]) {
```
Comment (ged-odoo) [inline src/store.ts]: maybe you could assign componentId above this line, it is used 3 times now...

Comment (seb-odoo) [reply src/store.ts]: I was thinking that too but I already pushed the commit... I'll change it then.

Comment (seb-odoo) [reply src/store.ts]: Rebased, added componentId, and replaced all the occurrence in the function

**Pull request merged**